ui-ngx/src/app/modules/home/components/widget/lib/maps/leaflet-map.ts

```diff
@@ -410,10 +413,15 @@ export default abstract class LeafletMap {
     }
 
     convertToCustomFormat(position: L.LatLng): object {
-        return {
-            [this.options.latKeyName]: position.lat % 90,
-            [this.options.lngKeyName]: position.lng % 180
-        };
+      if(position.lng > 180){
```

Comment:
where is check for lat?

Reply:
On maps, user can't set a point outside the map area in lat. This is probably a limitation at the leaflet level.